### PR TITLE
Fixes some Annoyances and socials

### PR DESCRIPTION
**Pull request by wout-codes**
Links:
1. `https://studyandanswers.com/mathematics/question7218761`
2. `https://smartanswer-ph.com/english/why-does-the-statue-of-liberty-look-1985986`
3. `aiseesoft.com`
4. `sejda.com`

### CODE REVIEW

**Comment by wout-codes** [comment]
`sejda.com` 

<details>

image
</details>

**Comment by wout-codes** [comment]
`https://studyandanswers.com/mathematics/question7218761`


<details>

image
</details>

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! placeholders in studyandanswers.com
+studyandanswers.com#?#div.banner:contains(-gpt-ad-)
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:5]: Is something useful hidden by `.banner`?
I see only one big leftover
`studyandanswers.com##.banner-top-comp`

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -38,6 +38,8 @@ jeotermalhaberler.com,govoritmoskva.ru,namemc.com,kevinpurcell.org,015.by,airkul
 !
 indirimdemarket.com#$##social-media-modal { display: none!important; }
 indirimdemarket.com#$#body { overflow: visible!important; }
+aiseesoft.com###share-list
+sejda.com##ul.social-list
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:42]: cannot find. Is tag required?

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: I don't trust this site. There are a few banners below which may be incorrectly blocked.

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:42]: Check my screenshot and it requires you to upload a PDF file then make some changes, apply changes, you should see socials. For Aiseesoft you gotta visit a site.

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@wout-codes Hi. Could you please write something to filters@adguard.com?

**Comment by wout-codes** [comment]
> @wout-codes Hi. Could you please write something to [filters@adguard.com](mailto:filters@adguard.com)?

Hello! Okay, sure.

**Comment by wout-codes** [comment]
@Alex-302 Did you say anything in my the E-Mail. Can you tell me here please? I don't like to open E-Mails.

**Comment by Alex-302** [comment]
@wout-codes yes, replied)

**Comment by wout-codes** [comment]
> @wout-codes yes, replied)

Can you tell me what you replied there? I don't really open Emails.

**Pull request merged**